### DISCUSSION TITLE
What means “Works With: Universal” for Visual Studio Code extensions

### DISCUSSION THREAD

**Discussion by alefragnani** [microsoft/vsmarketplace#297] [Q&A] [answered]
I just noticed a new _fixed_ info for Visual Studio Code extension’s page. It says:

—-
**Works With**
Universal
—-

What’s the meaning of this info and what I, as an extension developer, should do to properly update it (if necessary)?

Thank you

PS: Looking at _regular_ Visual Studio side, the same **Works With** info seems to show the Visual Studio release which the extension is compatible, but because VSCode doesn’t follow a similar release cycle, I suppose this is not the case.

**Comment by prashantvc** [comment, marked as answer]
We have recently enabled [platform specific extensions](https://code.visualstudio.com/api/working-with-extensions/publishing-extension#platformspecific-extensions). That is, you can upload VSIX targeting specific OS/CPU combination.
Universal means the extension will work on all operating systems and CPU.

For example, [C# extension](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.csharp) multiple download option targeting different OS
